OpenEdXMobile/res/layout/fragment_course_outline.xml

```diff
@@ -7,6 +7,11 @@
     android:orientation="vertical"
     android:splitMotionEvents="false">
 
+    <android.support.v4.widget.SwipeRefreshLayout
+        android:id="@+id/swipe_container"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
     <ListView
```

Comment:
Listview's indentation will change cuz now its nested within the `SwipeRefreshLayout`.

```suggestion
        <ListView
            android:id="@+id/outline_list"
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            android:choiceMode="singleChoice"
            android:divider="@null"
            android:dividerHeight="0dp"
            android:visibility="gone"
            tools:visibility="visible" />
```

p.s. trying out the suggest code feature on Github 😉

Reply:
This is remaining.